src/target.zig

```diff
@@ -323,23 +323,26 @@ pub fn selfHostedBackendIsAsRobustAsLlvm(target: std.Target) bool {
 }
 
 pub fn supportsStackProbing(target: std.Target) bool {
-    return target.os.tag != .windows and target.os.tag != .uefi and
-        (target.cpu.arch == .x86 or target.cpu.arch == .x86_64);
+    return switch (target.cpu.arch) {
+        .x86, .x86_64 => target.os.tag != .windows and target.os.tag != .uefi,
+        .spirv32, .spirv64 => false,
+        else => true,
```

Comment:
To preserve the original logic:
```suggestion
        else => false,
```